util/libssl.num

```diff
@@ -462,3 +462,4 @@ SSL_SESSION_set_protocol_version        462	1_1_1	EXIST::FUNCTION:
 OPENSSL_cipher_name                     463	1_1_1	EXIST::FUNCTION:
 SSL_alloc_buffers                       464	1_1_1	EXIST::FUNCTION:
 SSL_free_buffers                        465	1_1_1	EXIST::FUNCTION:
+DTLS_set_timer_cb                       466	1_1_1	EXIST::FUNCTION:
```

Comment:
Also in case you want to squash the commit, it is fine.
Maybe make your commit message describe your patch,
not the review process.